docs/devenv_rhel8.md

```diff
@@ -103,9 +103,11 @@ sudo subscription-manager repos \
 sudo dnf localinstall -y ~/microshift/packaging/rpm/_rpmbuild/RPMS/*/*.rpm
 ```
 
-Download the OpenShift pull secret from the https://console.redhat.com/openshift/downloads#tool-pull-secret page. Copy it to `/etc/crio/openshift-pull-secret` and update its file permissions so CRI-O can use it when fetching container images:
+Download the OpenShift pull secret from the https://console.redhat.com/openshift/downloads#tool-pull-secret page. Copy it to `/etc/crio/openshift-pull-secret` and update its file permissions so `CRI-O` can use it when fetching container images. The `CRI-O` service should be restarted to pick up the change and enabled to run by default after reboot.
 ```bash
 sudo chmod 600 /etc/crio/openshift-pull-secret
+sudo systemctl restart crio
```

Comment:
We shouldn't restart crio here as starting the microshift.service later should handle that and additionally doing it here may mask bugs related to missing restarts.-